test/unit/math/opencl/opencl_context_test.cpp

```diff
@@ -4,6 +4,13 @@
 #include <vector>
 #include <fstream>
 
+TEST(MathGpu, to_size_t_test) {
+  cl::size_t<3> a = stan::math::opencl::to_size_t<3>({1, 2, 3});
```

Comment:
@syclik would this do? Or were you thinking of something more complicated?

Reply:
Yes, this will do! If, for some unseen reason, someone decides to mess with the implementation, this will prevent that from happening.